web/src/main/java/com/bazaarvoice/emodb/web/auth/DefaultRoles.java

```diff
@@ -40,7 +40,7 @@
             Permissions.createSorTable(NON_SYSTEM_TABLE),
             Permissions.setSorTableAttributes(NON_SYSTEM_TABLE)),
 
-    // sor_standard with excluding pii tables/placements.
+    // sor_standard with anyCharExcept pii tables/placements.
```

Comment:
I'm only marking this here, though there may be more.  Looks like there was an over-ambitious search-and-replace of all instances of the word "excluding" with "anyCharExcept".  Search for uses of "anyCharExcept" and change them back in comments as appropriate.